src/controller/python/test/test_scripts/base.py

```diff
@@ -446,6 +447,7 @@ async def TestAddUpdateRemoveFabric(self, nodeid: int):
             self.logger.error("IssueNOCChain failed to generate valid cert chain for UpdateNOC with new node ID")
             return False
         updateNocResponse = await tempDevCtrl.SendCommand(nodeid, 0, Clusters.OperationalCredentials.Commands.UpdateNOC(chainForSecondUpdateNOC.nocBytes, chainForSecondUpdateNOC.icacBytes))
+        tempDevCtrl.ExpireSessions(nodeid)
```

Comment:
Could you fix the TODO below and add logic to read back data using the old node ID? This should fail relatively immediately since it'll attempt to establish a new CASE session that will immediately fail on Sigma1 (or Sigma2?) due to the lack of matching certs.

Reply:
An attempt to do so correctly failed as expected, but it took a while to fail due to the mDNS discovery that needs to timeout. Given that we don't want these tests to take super long and we don't have test infra support to actually modally enable/disable long-running tests in the Python testing framework, let's omit this for now.